src/index.html

```diff
@@ -3,12 +3,56 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0,
+          maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__menu">
+          <li class="header__li">
+            <a href="/" class="header__link is-active">Apple</a>
```

Comment:
Put some mock links in your all of your href attributes, e.g. `#apple`
```suggestion
            <a href="#apple" class="header__link is-active">Apple</a>
```